core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -163,6 +164,18 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest with Dou
     return Array(descaledsmallCoeff, originalsmallCoeff, descaledbigCoeff, orginalbigCoeff)
   }
 
+  def getFeatureMomentsAndCard(inputModel: FeatureLike[Prediction],
```

Comment:
I think it might be easier to read in the tests below if you make this return a `Map[String, FeatureDistribution]` where the key is the feature name - that way we know which distribution goes with which input feature.

You could make it like:
```def getFeatureMomentMap(inputModel: FeatureLike[Prediction], df: DataFrame): Map[String, FeatureDistribution] = {
    val workFlow = new OpWorkflow().setResultFeatures(inputModel).setInputDataset(df)
    val worflowWithRFF = workFlow.withRawFeatureFilter(
      trainingReader = Some(workFlow.getReader()),
      scoringReader = None
    )
    val model = worflowWithRFF.train()

    val insights = model.modelInsights(inputModel)
    insights.features.map(f => f.featureName -> f.distributions.head).toMap
  }```